cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -85,7 +85,6 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
                              npcConwayGenesisFileHash
                            }
                            npc@NodeHardForkProtocolConfiguration {
-                            npcExperimentalHardForksEnabled,
```

Comment:
Rather than delete this can you comment it out. We will have to re-enable it in the future.